src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Color color = colorSupplier.getRandomColor();
+        int maxBallNumber = 100;
```

Comment:
create constant

Reply:
Same issue here 
[ERROR] C:\Users\andre\IdeaProjects\jv-lottery\src\main\java\core\basesyntax\Lottery.java:11:19: Abbreviation in name 'MAX_BALL_VALUE' must contain no more than '2' consecutive capital letters. [AbbreviationAsWordInName]

Reply:
Ok, i figured it out. Constants are static final fields and belong to a class, not a variable or method.